### ISSUE TITLE
CGAL issue with planar advanced brep

### ISSUE THREAD

**Issue by Moult** [IfcOpenShell/IfcOpenShell#5484]
I think related to 2557fcf628d338b64b7023cb78e0f9db1d0b5182 there is now a regression with most of the coverings missing in the LA-p.ifc model I sent in #5444. For example this GlobalId: `39P2cY9rj36R7I8XWC4AaW`

**Comment by aothms** [comment]
So, many layers of bugs, but thanks for the report. Should be ok again.

**Comment by Moult** [comment]
Tested with latest bot build, there still seem to be a few missing elements:

`3zq1FD3n53ewi0xnLhNS1u` and `3jE2aI$RDBOPmWMmw0838I` and `2wIasairn5XPF3syGHDjKq`

**Comment by aothms** [comment]
Yeah technically maybe not wrong, because cgal does output something, just occt would output more, but you don't know at that time.

![afbeelding](https://github.com/user-attachments/assets/d52cf56e-dd61-4a0e-a56b-ed6e2a2bc7dc)

I've reshuffled a bit the exception handling (@Andrej730 have a cursory glance as well).

Btw these adv breps don't render that well in occt either:

![afbeelding](https://github.com/user-attachments/assets/8cbf672c-cbc8-4f83-a1e4-0835bb13d549)

![afbeelding](https://github.com/user-attachments/assets/bb78c891-26bd-4215-bd04-fad4f00cf803)
